src/App.js

```diff
@@ -277,7 +277,6 @@ const App = () => {
       theme={designerTheme}
       themeMode={colorMode}
       dir={rtl ? 'rtl' : undefined}
-      style={{ height: '100%' }}
```

Comment:
I seem to recall this is needed for reasons I can't remember. :(

Reply:
I do like that approach better, I will give it a try

Reply:
Hrmm, this change works everywhere well except the my designs area (it's all collapsed - I can't see my thumbnails). Looking more into it.